dbt_subprojects/hourly_spellbook/models/_project/balancer/erc4626_tokens/ethereum/balancer_v3_ethereum_erc4626_token_mapping.sql

```diff
@@ -1,7 +1,7 @@
 {{
     config(
         schema = 'balancer_v3_ethereum',
-        alias = 'erc4626_tokens_mapping', 
+        alias = 'erc4626_token_mapping', 
```

Comment:
Make sure fix the schema and downstream models are consistent.

Reply:
all good!